pages/nft-book-store/status/[classId].vue

```diff
@@ -637,6 +654,11 @@
       />
     </template>
 
+    <EditISCNMetadataModal
+      v-model="showModal"
+      :class-id="classId"
```

Comment:
Is there anything that needs to be refreshed in this UI after saving new ISCN?

Reply:
No need, the content of the new ISCN will not appear on this page. Opening the modal will also fetch the latest data.